### DISCUSSION TITLE
Extrach headings and subheadings

### DISCUSSION THREAD

**Discussion by bansalsahab** [jsvine/pdfplumber#1200] [Q&A]
I want to extract the heading and subheadings from an pdf document with page number in which it is .

**Comment by jsvine** [comment]
In many cases, headings in a PDF are not explicitly defined but rather implicitly indicated through their styling (font name, font size, color, etc.). You can identify those via `page.chars` (for character-level data) or `page.extract_words(extra_attrs=["fontname", "size"])` (for an attempt to auto-identify individual words).

**Reply by bansalsahab** [reply]
Ya i did it by first convert the pdf into html format and then parsed the headings from that
